regression-test/suites/correctness/test_pushdown_pred_to_view.groovy

```diff
@@ -54,7 +54,7 @@ The same resolve error occurs when re-analyze v2.
      """
 
      qt_sql """
-         select * from ${viewName} as v1 join ${viewName} as v2 on v1.id=v2.id and v1.id>0;
+         select * from ${viewName} as v1 join ${viewName} as v2 where v1.id=v2.id and v1.id>0;
```

Comment:
After fix this bug, we found that there is another bug hidden by this bug.
If we use `on` instead of `where`, the planner push `v1.id=v2.id` down to scan node.
That is why we modified the regression test temporarily.
We will fix it latter.

Reply:
add a TODO to this regression test?

Reply:
it is better to add another test case for `join ... on` after we fixed the second bug.